docs/index.rst

```diff
@@ -0,0 +1,155 @@
+
+1. *Count* : No of people detected.
+
+2. *Coordinates* : the coordinates of the bounding box detected (2 points each having x and y respectively)
+
+3. *Confidence* : The probability by which the model is sure that the detected object is a person.
+
+
+
+|config1|
+
+  - **TFlite Model File**:
+                            This is the name of the tflite model file that should be placed in
+                            python/fledge/plugins/south/person_detection/model directory.
+                            Its default value is detect_edgetpu.tflite.
+                            If a Coral Edge TPU is not being used, the file name will be different (i.e. detect.tflite).
+
+  - **Labels File**:
+                            This is the name of the labels file that was used when training the above
+                            model, this file should also be placed in same directory as the model.
+
+
+  - **Asset Name**:
+                           The name of the asset used for the readings generated by this plugin.
+
+  - **Enable Edge TPU**:
+                           Indicates whether to use edge TPU for inference.
+                           If you don't want to use Coral Edge TPU then disbale this configuration parameter.
+                           Also ensure to change the name of the model file to detect.tflite if disabled.
+                           Default is set to enabled.
+
+  - **Minimum Confidence Threshold**:
+                           The detection results from the model will be filtered out, if the score is below this value.
+
+  - **Source**:
+                          Either use a stream over a network or use a local camera device.
+                          Default is set to stream.
+
+  - **Streaming URL**:
+                         The URL of the RTSP stream, if stream is to be used. Only RTSP streams are supported for now.
+
+  - **Open CV Backend**:
+                         The backend required by Open CV to process the stream, if stream is to be used.
+                         Default is set to ffmpeg. Note gstreamer is also supported but not used.
+
+  - **Streaming Protocol**:
+                         The protocol over which live frames are being broad casted over the network, if stream is to be used.
+                         Open CV uses tcp by default. Also Open CV can use other protocols as well.
+                         The plugin does not support tcp.
+                         Default is set to udp.
+
+  - **Camera ID**:
+                        The number associated with your video device. See /dev in your filesystem you will see video0 or video1.
+                        It is required when source is set to camera.
+                        Default is set to 0.
+
+  - **Enable Detection Window**:
+                        Show detection results in a native window.
```

Comment:
What is "native window"?

Reply:
Opencv's own gui display. the window that appears when we use **imshow**.